vsts/RunTestsOnEmulator.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# -e flag to force this command to run on what should be the only emulator active on this machine
+echo 'Installing APKs on emulator'
+adb -e install -r iot-e2e-tests/android/app/build/outputs/apk/debug/app-debug.apk
+adb -e install -r iot-e2e-tests/android/app/build/outputs/apk/androidTest/debug/app-debug-androidTest.apk
+
+#List instrumentation classes, for logging purposes only
+echo 'Listing available instrumentations:'
+adb -e shell pm list instrumentation
+echo ''
+
+annotationString="com.microsoft.azure.sdk.iot.android.helper.${TEST_GROUP_ID}"
+echo 'Running android tests with annotation'
+echo $annotationString
+
+#Return code from adb shell isn't returned as one would expect. Need to capture output logs and analyze them to determine if this test run was a success or not
```

Comment:
whoa!

Reply:
Yeah, I'm not happy about this. I don't have another way though